spid_cie_oidc/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.8.15"
+__version__ = "0.8.16"
```

Comment:
```suggestion
__version__ = "0.9.0"
```

what about this @rglauco ?
It seems that we're introducing several, even if "missing", features

Reply:
agree on that